src/Sulu/Bundle/PageBundle/Preview/PageObjectProvider.php

```diff
@@ -70,6 +70,9 @@ public function setValues($object, $locale, array $data)
             ->enableMagicCall()
             ->getPropertyAccessor();
 
+        $object->setLocale($locale);
+        $object->setOriginalLocale($locale);
+
```

Comment:
I think this will not work for the ContentBundle: https://github.com/sulu/SuluContentBundle/blob/8435b81422f708be0d7733c91526b981a9fe8a96/Content/Infrastructure/Sulu/Preview/ContentObjectProvider.php#L144

There we are loading the locale from the cached document. And if we not restart the preview via the frontend still the old `locale` is in the `document`. A setter is bad as we need to load the document by the correct locale from the database.

Also I think with the current way things like `settings` e.g.: `author` is still false in the cached document as they are not from `setValues` given. So only data is given from the current tab not the other tabs.